src/core/default/commands/function/logs.py

```diff
@@ -33,22 +33,45 @@ def add_arguments(self, parser: ArgumentParser) -> None:
             "--tail", action="store_true", help="show the tail of the logs"
         )
         parser.add_argument(
-            "--number",
-            action="store_true",
+            "--limit",
+            type=int,
+            nargs='?',
+            default=10000,
             help="number of events to show. Must be used with --tail.",
         )
+        parser.add_argument(
+            "--query",
+            nargs='+',
+            help="retrieve information base on a cloudWatch query",
+        )
+        parser.add_argument(
+            "--start_time",
+            type=lambda d: datetime.datetime.strptime(d, '%Y-%m-%d-%H:%M:%S'),
+            help="set the start_time when use the query option, must be on the %Y-%m-%d-%H:%M%S format",
```

Comment:
When I run the command with the `-h` flag, it throws an error (`unsupported format character 'Y' (0x59) at index 62`) about the date formatting in this help string. If you remove the `%` character from the help string the `-h` flag works properly.